<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoListView</class>
 <widget class="QDialog" name="VideoListView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="memberLabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>511</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cochin</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="video2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>40</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="audio6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="video3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="videoView7" native="true">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>520</y>
     <width>241</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>30</y>
     <width>261</width>
     <height>3</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="switchAllRemoteVideo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>0</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>全部静画</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>280</y>
     <width>251</width>
     <height>3</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="audio5">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>290</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="video0">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="videoView5" native="true">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>280</y>
     <width>261</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="audio2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>40</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="video4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>290</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="videoView2" native="true">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>30</y>
     <width>261</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="video1">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar7">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>520</y>
     <width>241</width>
     <height>3</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="audio7">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>530</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="videoView0" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>251</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar5">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>280</y>
     <width>261</width>
     <height>3</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="audio1">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="screenCaptureView" native="true">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>520</y>
     <width>261</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="audio4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>290</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="switchAllRemoteAudio">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>0</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>全部静音</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar1">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>241</width>
     <height>3</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="audio3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="videoView1" native="true">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>241</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="videoView4" native="true">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>280</y>
     <width>241</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="switchDashBoardButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>0</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>仪表盘</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>280</y>
     <width>241</width>
     <height>3</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="video5">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>290</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="video7">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>530</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="audio0">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="videoView3" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>280</y>
     <width>251</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="videoView6" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>520</y>
     <width>251</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>520</y>
     <width>251</width>
     <height>3</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="video6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>530</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar0">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>251</width>
     <height>3</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <zorder>videoView6</zorder>
  <zorder>videoView7</zorder>
  <zorder>videoView5</zorder>
  <zorder>videoView4</zorder>
  <zorder>videoView3</zorder>
  <zorder>videoView2</zorder>
  <zorder>videoView1</zorder>
  <zorder>videoView0</zorder>
  <zorder>memberLabel</zorder>
  <zorder>video2</zorder>
  <zorder>audio6</zorder>
  <zorder>video3</zorder>
  <zorder>progressBar2</zorder>
  <zorder>switchAllRemoteVideo</zorder>
  <zorder>progressBar3</zorder>
  <zorder>audio5</zorder>
  <zorder>video0</zorder>
  <zorder>audio2</zorder>
  <zorder>video4</zorder>
  <zorder>video1</zorder>
  <zorder>progressBar7</zorder>
  <zorder>audio7</zorder>
  <zorder>progressBar5</zorder>
  <zorder>audio1</zorder>
  <zorder>audio4</zorder>
  <zorder>switchAllRemoteAudio</zorder>
  <zorder>progressBar1</zorder>
  <zorder>audio3</zorder>
  <zorder>switchDashBoardButton</zorder>
  <zorder>progressBar4</zorder>
  <zorder>video5</zorder>
  <zorder>video7</zorder>
  <zorder>audio0</zorder>
  <zorder>progressBar6</zorder>
  <zorder>video6</zorder>
  <zorder>progressBar0</zorder>
  <zorder>screenCaptureView</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
